packages/edit-site/src/components/sidebar-navigation-screen-pattern/template-part-navigation-menu-list.js

```diff
@@ -10,7 +10,7 @@ import TemplatePartNavigationMenuListItem from './template-part-navigation-menu-
 export default function TemplatePartNavigationMenuList( { menus } ) {
 	return (
 		<ItemGroup className="edit-site-sidebar-navigation-screen-template-part-navigation-menu-list">
-			{ menus.map( ( menuId ) => (
+			{ [ ...new Set( menus ) ].map( ( menuId ) => (
```

Comment:
Is this change related to the title rendering fix?

If we need to ensure that `menus` are a unique set, then this probably has to be done higher in the tree, maybe in `useNavigationMenuContent` or `getBlocksOfTypeFromBlocks`?

Reply:
Yes I don't see a new for `Set` here unless I missed something...?